css/style-navig.css

```diff
@@ -0,0 +1,140 @@
+/* -- Slideout Sidebar -- */
+
+.slideout-sidebar {
+  position: fixed;
+  top: 0;
+  left: -190px;
+  z-index: 0;
+  width: 150px;
+  height: 100%;
+  padding: 20px;
+  background-color: #2121218f;
+  transition: all 300ms ease-in-out;
+}
+
+.slideout-sidebar ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.slideout-sidebar ul li {
+  cursor: pointer;
+  padding: 18px 0;
+  border-bottom: 1px solid rgba(244,244,244,0.4);
+}
+
+.slideout-sidebar ul li:last-child {
+  border-bottom: 0;
+}
+
+/* -- Menu Icon -- */
+
+#menu-toggle {
+  display: none;
+}
+
+.menu-icon {
+  position: absolute;
+  top: 18px;
+  left: 30px;
+  font-size: 24px;
+  z-index: 1;
+  transition: all 300ms ease-in-out;
+}
+
+
+
+/*-- The Magic --*/
+#menu-toggle:checked ~ .slideout-sidebar {
+  left: 0px;
+}
+
+#menu-toggle:checked + .menu-icon {
+  left: 210px;
+}
+
+#menu-toggle:checked ~ .content-container {
+  padding-left: 190px;
+}
+
+/* -- Media Queries -- */
+@media (max-width: 1280px){
+  .slideout-sidebar{
+    display: block;
+  }
+  
+  .menu-icon{
+    display: block;
+  }
+
+  .header-menu{
+    display: none;
+  }
+
+  a:-webkit-any-link {
+    color: #fff;
+    cursor: pointer;
+  }
+}
+
+@media (max-width: 991px) {
+  
+  .content-container {
+    max-width: 480px;
+  }
+  
+}
+
+@media (max-width: 767px) {
+  
+  .content-container {
+    margin: 30px auto 0;
+  }
+  
+  #menu-toggle:checked ~ .content-container {
+    padding-left: 0;
+  }
+  
+  .slideout-sidebar ul {
+    text-align: center;
+    max-width: 100px;
+    margin: 30px auto 0;
+  }
+  
+  .menu-icon {
+    left: 30px
+  }
+  
+  #menu-toggle:checked ~ .slideout-sidebar {
+    width: 55%;
+  }
+  
+  #menu-toggle:checked + .menu-icon {
+    left: 47%;
+    color: #fafafa;
+  }
+  
+  @media screen  
```

Comment:
you have `@media` query inside `@media` query. Please, be careful and patient with code formatting